example/lib/components/buttons.dart

```diff
@@ -130,6 +132,29 @@ class _ButtonsViewState extends State<ButtonsView> {
           ),
           const SizedBox(height: 16.0),
 
+          // Collapsible Button
```

Comment:
Why do we have a CollapsibleButton comment over a Secondary Outlined Button?

Reply:
ohhh, maybe by mistake